src/locales/en_US.js

```diff
@@ -691,7 +692,7 @@ const strings = {
   fio_get_reg_info_already_sent_err_msg: 'Registration request is already sent',
   fio_register_address_err_msg: 'There was an issue registering an address. Please try later.',
   fio_error_invalid_address: 'Invalid FIO address',
-  fio_add_bundles_err_msg: 'There was an issue purchasing new bundles. Please try again later.',
+  fio_add_bundles_err_msg: 'There was an issue purchasing more bundled transactions. Please try again later.',
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name